README.md

```diff
@@ -1,10 +1,8 @@
 [![Chat at https://gitter.im/python/typing](https://badges.gitter.im/python/typing.svg)](https://gitter.im/python/typing?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Static Typing for Python
-========================
+# Static Typing for Python
```

Comment:
Oops, I think this was some markdown autoformatter. Let me know if you'd like me to undo it.